Dockerfiles/midstream/run_tests.py

```diff
@@ -53,32 +58,24 @@ def test_for_extract_and_validate_bundle_image(self):
         # check if IMAGE_TO_TEST is defined, return exit_code 102 in case it's not
         if (self.image_to_test is None):
             print("Environment variable IMAGE_TO_TEST not set! Stopping the tests.")
-            with open(".errormessage", 'w') as error_msg:
+            with open("/tmp/.errormessage", 'w') as error_msg:
                 print("Result code: 102 Error message: Environment variable IMAGE_TO_TEST not set!", file=error_msg)
             exit_code = 102
             return exit_code
-        if (self.operator_dir != '/project/operator_dir/' or
-           self.operator_work_dir != '/project/test_operator_work_dir/' or
-           self.playbooks_dir != '/project/operator-test-playbooks/' or
-           self.umoci_bin_path != '/usr/local/bin/umoci' or
-           self.work_dir != '/project/output/'):
-            print("Environment variable misconfigured!")
-            with open(".errormessage", 'w') as error_msg:
-                print("Result code: 103 Error message: Environment variable was not set correctly!", file=error_msg)
-            exit_code = 103
-            return exit_code
         result = self.run_extract_operator_bundle()
+        print(result.stdout)
         if (result.returncode != 0):
             print("Ansible playbook extract-operator-bundle.yml failed with result code: %s , see the file .errormessage for more info." % result.returncode)
-            with open(".errormessage", "w") as error_msg:
+            with open("/tmp/.errormessage", "w") as error_msg:
```

Comment:
last time you told; it has to be in home directory

Reply:
Yes. It was. However, In order to bypass the permission issue we are currently facing right now This approach was working.

Reply:
k8s/(jenkins plugin for k8s) always creates shared volume with homedir where default k8s user can write

If midstream has requirement to write file into HOMEDIR than we have to write into HOMEDIR.

Reply:
Resolved by recent update.